packages/app/src/systems/Chains/fuel/utils/logo.ts

```diff
@@ -0,0 +1,3 @@
+import fuelLogoSrc from '../../../../../public/fuel-logo.svg';
+
+export { fuelLogoSrc };
```

Comment:
this whole file was removed on PR with deposit flow, do you need to merge with main? or maybe was something weird when merging.


I removed all usages, and also this file. instead replaced everything with `<FuelLogo` as you suggested